optic14n.gemspec

```
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/alphagov/optic14n"
   spec.license       = "MIT"
 
-  spec.required_ruby_version = "> 2.7.0"
+  spec.required_ruby_version = "> 3.0.5"
```

```suggestion
  spec.required_ruby_version = "> 3.0"
```

This should be reduced to the major version